demo/snakemake.slurm.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Launches a gambl snakefile of your choice on numbers and sets the conda prefix
+
+# Usage: ./snakemake.number.sh <snakefile.smk> <target_rule> <unix_group> <snakemake_flags>
+
+# Example: ./snakemake.numbers.sh src/snakemake/manta.smk all gambl --rerun-incomplete
```

Comment:
Modify to `./snakemake.slurm.sh`

Reply:
done. All mentions of "numbers" now refer to slurm instead. I also removed any mention of the unix group because that option was removed.